loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -864,6 +866,27 @@ export class GLTFLoader implements IGLTFLoader {
             const accessor = ArrayItem.Get(`${context}/attributes/${attribute}`, this._gltf.accessors, attributes[attribute]);
             promises.push(this._loadVertexAccessorAsync(`/accessors/${accessor.index}`, accessor, kind).then((babylonVertexBuffer) => {
                 babylonGeometry.setVerticesBuffer(babylonVertexBuffer, accessor.count);
+                if (babylonVertexBuffer.getKind() === VertexBuffer.PositionKind) {
+                    postProcesses.push(() => {
```

Comment:
I don't think this is a good idea. Some day, we will remove the underscore from the _loadVertexDataAsync function so that it will be part of the contract for glTF loader extension (currently because of the underscore, it is considered experimental). I don't think adding this kind of mechanism should be part of this contract.

Reply:
It seems like Geometry has bounding info. I wonder if it makes sense to update the geometry's bounding box info with the info from min/max and then make sure the code is taking advantage of it?

I also see things like the mesh taking the world matrix into account for the bounding info. Does this code work when there is a non-identity transform on the mesh?

Reply:
Regarding the bounding info of the `Geometry class`, it is in fact not used in the class.

So, I have added the possibility to use this bounding info as the bounding info applied to meshes sharing the geometry and have updated the GLTF loader to use this new capability.

Let me know if it's ok for you.

Note: I have checked that using non identity transforms does work with the new code.

Reply:
Nice. I like this much better. Is the flag on the geometry necessary? Can we use the presence of the boundingInfo? @sebavan ?

Reply:
I have updated the code. Let's wait for @sebavan inputs.

Note that `Geometry._boundingInfo` is never set by the class, and the only one location it is read from is here (before the code update):
```
        if (this.isReady()) {
            this._applyToMesh(mesh);
        }
        else {
            mesh._boundingInfo = this._boundingInfo;
        }
```
It is at the end of `applyToMesh`.